test/numerical/numerical.def

```diff
@@ -1,5 +1,5 @@
-EXPORTS 
-run_main_graph
-omInputSignature
-omOutputSignature
-
+EXPORTS
```

Comment:
Can you double check that these changes are made to all of the ".def" files? Thanks

Reply:
Done for other .def files. Thanks!